styles/settings/_typography.scss

```diff
@@ -1,18 +1,18 @@
 /* ==========================================================================
    Stacks
    ========================================================================== */
-$typo-stack-base: 'Whitney SSm A', 'Whitney SSm B', sans-serif !default;
+$typo-stack-base: 'IBM Plex Sans', sans-serif !default;
```

Comment:
Where is this being sourced from?

Reply:
At the moment this comes from the app codebase. Do you think its worth moving the font files into gather-ui? We can then still include them for old accounts using the app

Reply:
How does the library handle this currently? (check `/.storybook/config.js`).

------------

I don't think we need to build anything for including fonts elsewhere but the library itself should include fonts so when people work on it they're using the correct typeface 👍

Reply:
It points to a typography URL but we will be self hosting IBM-plex. The reason I say we'd have to include it elsewhere is because not everyone has the new webapp css included on there account yet